dlf/ext_tables.php

```diff
@@ -221,7 +221,9 @@
 t3lib_extMgm::addStaticFile($_EXTKEY, 'typoscript/', 'Basic Configuration');
 
 // Register plugins.
-t3lib_div::loadTCA('tt_content');
+if (version_compare(TYPO3_branch, '6.1', '<')) {
```

Comment:
minumum TYPO3 version specified for this ext is 6.2. Why not just removing these lines completely?